po/README.md

```diff
@@ -0,0 +1,27 @@
+# Translations generated by Launchpad
+
+## How to sync translations from Launchpad?
+- Make sure `GOPATH` is set and `GOINVOKEFLAGS="-mod=readonly"`.
+- Run `update-pot` to generate latest translation template (`po/snappy.pot`).
+- Import generated translation template to [the snappy entry][templates].
+- Wait until import is approved. You can monitor status in the [import queue][import-queue].
+- Export [latest translations][export] (download link will be sent to your email).
+- Extract archive: `tar -xvf launchpad-export.tar.gz`.
+- Create new directory to store processed translations `mkdir processed`.
+- `for file in snappy-*; do msgattrib --no-obsolete --translated --output-file="processed/${file#snappy-}" "$file"; done;`
```

Comment:
Dropping straight to a script like this seems a little jarring. Perhaps prefix this line with something like:
- Run the following to remove the `snappy-` prefix, remove obsolete `#-` messages (`--no-obsolete`) and exclude untranslated messages (`--translated`):
 `for file in snappy-*; do msgattrib --no-obsolete --translated --output-file="processed/${file#snappy-}" "$file"; done;`

Reply:
Great point, Thank you!